docs/modules/ROOT/pages/lite-members.adoc

```diff
@@ -0,0 +1,55 @@
+= Deploying Lite Members
```

Comment:
I think we should add some information at https://docs.hazelcast.com/operator/latest/connect-outside-kubernetes related to lite members. WDYT?

Reply:
Now we are testing service per pod, and if it works the same, I think we shouldn't add anything to this doc.